app/views/bus_stops/by_sequence.haml

```diff
@@ -0,0 +1,13 @@
+.page-title Route #{@route.number} Bus Stops
+
+.container
+  - @collection.each do |direction, bsrs|
+    = direction + 'bound'
```

Comment:
I called `BusStopsRoute.pluck(:direction).uniq` and got `["South", "North", "West", "East", "Downward", "Upward"]`. Maybe "downwardbound" looks strange, but it doesn't really matter.

Reply:
Do we need the 'bound'?

Reply:
No. Let's go without it.